### PR TITLE
Fix loading in windows, url paths

### PR DESCRIPTION
**Pull request by francescalb**
This works for me now in both windows and linux.
from ontopy import get_ontology
onto = get_ontology('https://raw.githubusercontent.com/BIG-MAP/BattINFO/master/battinfo.ttl').load()**

Can you please check?

# Description:

## Type of change:

- [ ] Bug fix.
- [ ] New feature.
- [ ] Documentation update.

## Checklist:

This checklist can be used as a help for the reviewer.

- [ ] Is the code easy to read and understand?
- [ ] Are comments for humans to read, not computers to disregard?
- [ ] Does a new feature has an accompanying new test (in the CI or unit testing schemes)?
- [ ] Has the documentation been updated as necessary?
- [ ] Does this close the issue?
- [ ] Is the change limited to the issue?
- [ ] Are errors handled for all outcomes?
- [ ] Does the new feature provide new restrictions on dependencies, and if so is this documented?

### CODE REVIEW

File: ontopy/utils.py [hunk 1]
```diff
@@ -350,7 +350,7 @@ def load_uri(uri, dirname):
                 url = f"{baseuri}/{uri_as_str}"
             else:
                 url = os.path.join(baseuri if baseuri else dirname, uri_as_str)
-        url = normalise_url(url)
+        # url = normalise_url(url)
```
Comment (CasperWA) [inline ontopy/utils.py:353]: Wouldn't this create other issues, since the URL is now not normalized?

Comment (francescalb) [reply ontopy/utils.py:353]: Both Jesper and I tried to see if we found such cases, but did not. The reason why it was commented out and not deleted completely was for easy reversal (and then searching for a new solution for Windows) if needed.

**Comment by sygout** [comment]
I tried to run pytest on Windows and got: 5 failed, 17 passed, 2 skipped, 7 warnings

**Comment by francescalb** [comment]
> I tried to run pytest on Windows and got: 5 failed, 17 passed, 2 skipped, 7 warnings

Yes, I do to on windows. We need to set up a testing environment for windows. Since I (almost) never use windows for this I have not encoutnered them. I suggest making a separate issue for creating a windows testing environment though. And we just make sure that loading works in both windows and linux for now

**Comment by sygout** [comment]
On a fresh installation (new virtual environment and clean repository clone), the correction seems to work as test_load is working, the error present in master v0.3.1-19-g02b3c84 (FAILED test_load.py::test_load - urllib.error.HTTPError: HTTP Error 400: https://raw.githubusercontent.com/%5CBIG-MAP) does not appear anymore.

**Pull request merged**